scripts/eurostat/health_determinants/common/euro_stat.py

```diff
@@ -0,0 +1,328 @@
+sys.path.insert(1, _COMMON_PATH)
+# pylint: disable=wrong-import-position
+from common.replacement_functions import (split_column, replace_col_values)
+from common.sv_config import file_to_sv_mapping
+
+# For import util.alpha2_to_dcid
+_COMMON_PATH = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '../../../../'))
+sys.path.insert(1, _COMMON_PATH)
+
+from util.alpha2_to_dcid import COUNTRY_MAP
+# pylint: enable=wrong-import-position
+
+_FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_TMCF_TEMPLATE = ("Node: E:eurostat_population_{import_name}->E0\n"
+                  "typeOf: dcs:StatVarObservation\n"
+                  "variableMeasured: C:eurostat_population_{import_name}->SV\n"
+                  "measurementMethod: C:eurostat_population_{import_name}->"
+                  "Measurement_Method\n"
+                  "observationAbout: C:eurostat_population_{import_name}->geo\n"
+                  "observationDate: C:eurostat_population_{import_name}->time\n"
+                  "scalingFactor: 100\n"
+                  "value: C:eurostat_population_{import_name}->observation\n")
+
+
+class EuroStat:
+    """
+    EuroStat is a base class which provides common implementation for generating 
+    CSV, MCF and tMCF files.
+    """
+    # Below variables will be initialized by sub-class (import specific)
+    _import_name = ""
+    _mcf_template = ""
+    _sv_value_to_property_mapping = {}
+    _sv_properties_template = {}
+    _sv_properties = {}
+
+    def __init__(self, input_files: list, csv_file_path: str,
+                 mcf_file_path: str, tmcf_file_path: str) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._df = pd.DataFrame()
+
+    def _propety_correction(self):
+        None
+
+    def _sv_name_correction(self, sv_name: str) -> str:
+        None
+
+    def _parse_file(self, file_name: str, df: pd.DataFrame,
+                    import_name: dict) -> pd.DataFrame:
+        split_columns = df.columns.values.tolist()[0]
+        df = split_column(df, split_columns)
+
+        split_columns = split_columns.replace('isced97', 'isced11')\
+                .replace('geo\time','geo').replace('geo\\time','geo')\
+                .replace('time\geo','time').replace('time\\geo','time')\
+                .replace('quantile', 'quant_inc')
+
+        df.rename(columns={
+            'geo\time': 'geo',
+            'geo\\time': 'geo',
+            'time\geo': 'time',
+            'time\\geo': 'time',
+            'isced97': 'isced11',
+            'quantile': 'quant_inc'
+        },
+                  inplace=True)
+
+        if import_name == "alcohol_consumption":
```

Comment:
can the list of columns to be replaced be passe din as anarg for init? that way we can remove the import specific code from the common utility.

```
__init__{ ...column_replacements: dict)
   self._column_replacements = column_replacements


def __parse_file()
...
   for column_name, replacement in self._column_replacements:
     if column_name in df.columns:
         df.rename(columns = { column_name: replacement }, inPlace=true)
...
```